api/src/pcapi/routes/adage_iframe/serialization/logs.py

```diff
@@ -9,6 +9,10 @@ class AdageBaseModel(BaseModel):
     isFromNoResult: bool | None
 
 
+class VueTypeMixin(BaseModel):
```

Comment:
Ce n'est plus un mixin si ça hérite du `BaseModel`.
Les Mixin sont un peu comme l'équivalent des interfaces dans d'autres langages.
Ils/Elles sont généralement posées **avant** les classes de bases dans les déclarations pour s'assurer que les classes "terminales"  - `AdageBaseModel` dans notre cas - soient bien appelées.
```suggestion
class VueTypeMixin:
```

Reply:
Le problème est que si cette classe n'hérite pas de `BaseModel`, `vueType` n'est pas exporté.

Concrètement, si je lance la commande qui met à jour le client front sans cet héritage, il n'y a aucune différence. Dès que je le rajoute, le code front attendu est généré.